src/System.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/SqlAdapterUpdateBatch.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public class SqlAdapterUpdateBatch
+    { 
+        private static string tcpConnStr = DataTestUtility.TcpConnStr;
+        private static string tableName = "BatchDemoTable";
+        private static List<EventInfo> entities;
+
+        [Fact]
+        public void SqlAdapterTest()
+        {
+            try
+            {          
+                var createTableQuery = "IF NOT EXISTS (SELECT * FROM sysobjects WHERE name='BatchDemoTable' AND xtype='U')" +
+                    " CREATE TABLE [dbo].[" + tableName + "]([TransactionNumber][int] IDENTITY(1, 1) NOT NULL,[Level] [nvarchar] (50) NOT NULL," +
+                    "[Message] [nvarchar] (500) NOT NULL,[EventTime] [datetime]NOT NULL,CONSTRAINT[PK_BatchDemoTable] " +
+                    "PRIMARY KEY CLUSTERED([TransactionNumber] ASC)WITH(PAD_INDEX = OFF,STATISTICS_NORECOMPUTE = OFF, " +
+                    "IGNORE_DUP_KEY = OFF,ALLOW_ROW_LOCKS = ON,ALLOW_PAGE_LOCKS = ON,FILLFACTOR = 90) ON[PRIMARY]) ON[PRIMARY]";
+
+                using (var connection = new SqlConnection(tcpConnStr))
+                {
+                    var cmd = new SqlCommand(createTableQuery, connection);
+                    connection.Open();
+                    cmd.ExecuteNonQuery();
+                }
+                ExecuteNonQueries();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+                Console.WriteLine(ex.StackTrace);
+            }
+            finally
+            {
+                using (var connection = new SqlConnection(tcpConnStr))
+                {
+                    var dropTableQuery = "DROP TABLE IF EXISTS " + tableName;
+                    var cmd = new SqlCommand(dropTableQuery, connection);
```

Comment:
Wrap command in `using` so that it is disposed ?

Reply:
done